src/common/utilities/key-events.ts

```diff
@@ -0,0 +1,7 @@
+export enum KeyCodes {
```

Comment:
We don't need to create our own list of keycodes here, use `@angular/cdk/keycodes` instead.